Documentation/flexMigration/flex-to-csi-migration.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Migration
+weight: 4100
+indent: true
+---
+
+# Flex to CSI Migration
+
+The tool [persistent-volume-migrator](https://github.com/ceph/persistent-volume-migrator) aims to help migrate Flex rbd Volumes to Ceph-Csi.
+
+## Prerquisite
+
+This guide assumes you have created a Rook cluster with Flex
+1) Minimum ceph version should be Ceph-CSI supported.
+2) Volume should be in BOUND state and should **not** be in use.
+3) CSI storageClass should be created in which you want to migrate.
+   1) Example: CSI [storageclass](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/csi/rbd/storageclass.yaml).
+
+**NOTE**: The Migration procedure will come with a downtime; As we need to scale down the applications using the Volumes before migration.
+
+## Steps
+1) Create rbac required to run migrator pod
```

Comment:
Can we merge the rbac.yaml into the same manifest as the migrator pod? This keeps it to a single command to create the manifest.

Reply:
since we deleted the PR which was adding this file to rook repo. so mean to change this tool repo?

Reply:
updated